@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -205,7 +209,7 @@ export class SimpleTileGeometryLoader implements TileGeometryLoader {
         const tile = this.tile;
 
         // First time this tile is handled:
-        if (this.m_decodedTile === undefined && tile.decodedTile !== undefined) {
+        if (this.m_decodedTile === undefined && tile.decodedTile !== undefined && !this.isLoading) {
```

Comment:
Using ```!this.isLoading``` check here stops processing both techniques and tile geometry load, but this way we will need to _wait_ until timeout to cancel invisible or disposed tile loading. Other method would be to check this condition in the update section (as in prepareForRender) and if tile is no longer visible cancel timeout.
This may give us some small improvements so less timeouts may be queued while panning.